common/src/main/resources/docker/docker.bom

```diff
@@ -357,6 +364,46 @@ brooklyn.catalog:
             command: |
               cat ${DOCKER_CERT_PATH}/csr.pem
 
+  - id: docker-engine-with-resilience
```

Comment:
Whoever is reviewing more thoroughly: can you diff this text from that removed from `swarm/src/main/resources/swarm/swarm.bom` and let us know if anything has changed (ignoring white-space). And paste the diff as a response to this comment.